What’s up in Vietnam.md

```diff
@@ -65,58 +66,58 @@ UNION Col1, Null as Col2, Col3 FROM heating_devices.
 
-There are a lot of other databases matching different requirements. There are useful graph databases such as Neo4j (excellent database), column stores, spatial databases, probabilistic databases and many others.
+There are a lot of other databases matching different requirements. For example, we have very useful graph databases such as Neo4j (excellent database), column stores, spatial databases, probabilistic databases and many others.
 
 ## „OK, but I have relational data for sure and I need to use OO design of application because of some reason.”
 I will tell you a story.
 
-Once I had a task to create a wholly complicated ranking. Ranking should be working live for 27 000 active users. Every activity of the user should be reflected in scoreboard live. My first thought was to create some simplified CQRS with a system such as Apache Kafka calculating the ranking based on the stream of events.
+Once I had a task to create a  complicated ranking. Ranking should be handling for 27 000 live active users. Every activity of a user should be reflected on a live scoreboard. My first thought was to create some simplified CQRS with a system like Apache Kafka calculating the ranking based on the stream of events.
 
-Then my tech leader advised me: „don’t do it, use Sphinx”. I was totally lost, because Sphinx is a full-text search engine. How the hell deal with it to create ranking? Every row should have a full-text column. What to do with it? „Just leave it empty”, he said.
+Then my tech leader advised me: „don’t do it, use Sphinx”. I was completely lost, because Sphinx is a full-text search engine. How the hell is it supposed to create ranking? Every row should have a full-text column. What to do with it? „Just leave it empty”, he said.
 
-In the meantime, our team went to Neo4j conference which was organized by CD PROJECT company. They were creating a significant online card came - Gwent. I realized that they have to have even more significant problems with ranking, so I asked how they deal with it. „We’re using Sphinx” they answered. Then my tech lead asked them: „Why Sphinx? It’s a full-text search engine. How do you deal with a mandatory full-text field in a row?”. „Yeah, but it’s the simplest way to achieve it, and we just leave this mandatory field empty” they answered. I fooled myself.
+In the meantime, our team went to a Neo4j conference, hosted by CD Project game development studio. They were creating a large-scale online card came -- Gwent. I realized that they have to have even more problems with ranking, so I asked how they deal with it. „We’re using Sphinx” they answered. Then my tech lead asked them: „Why Sphinx? It’s a full-text search engine. How do you deal with a mandatory full-text field in a row?”. „Yeah, but it’s the simplest way to achieve it, and we just leave this mandatory field empty” they answered. 

```

Comment:
Why sentence "I fooled myself." disappeared?

Reply:
I had no idea what you meant by this sentence and story makes total sense without it.

Reply:
I've changed to `I made a fool of myself`.